pep-9999.rst

```diff
@@ -0,0 +1,89 @@
+PEP: 9999
+Title: Recording provenance of installed packages
```

Comment:
On second thought, maybe we shouldn't call it "provenance", which is a bit overloaded. Maybe "installation reports" is better. Better yet, maybe it could even be a new type of [in-toto attestation](https://github.com/in-toto/attestation).

Reply:
> On second thought, maybe we shouldn't call it "provenance", which is a bit overloaded. Maybe "installation reports" is better. 

Installation report can be confused with [pip's installation reports](https://pip.pypa.io/en/stable/reference/installation-report/).

> Better yet, maybe it could even be a new type of [in-toto attestation](https://github.com/in-toto/attestation).

Maybe we could add this directly to PEP as an example of usage, beside SBOM already mentioned.